### ISSUE TITLE
Enhancement: `/fail` messages for synchronous commands

### ISSUE THREAD

**Issue by jamshark70** [supercollider/supercollider#3898]
The Server Command Reference says:

> /fail
> An error occurred.
> 
> string	the name of the command
> string	the error message.
> other	(optional) some commands provide other information, for example a buffer index.
> 
> There was a problem. Sent only to the sender of the original message.

It does not specify that this should be only for asynchronous commands. However, it appears that informally this has been interpreted to apply only to asynchronous commands.

But, if (for example, as happened to me just today) incoming commands are accidentally evaluated out of order such that `/b_setn` precedes `/b_alloc`, certainly the user should get some feedback and I see no reason why this should not include an OSC reply.

`/b_setn` is an especially egregious case because it looks like scsynth doesn't even provide an error post at all!

```
Server.scsynth;
s.boot;

s.sendMsg(\b_setn, 0, 0, 3, 1, 2, 3);
```

Supernova tries to print a message, but a/ this does not include the buffer number and b/ currently there is a typo (#3897) that causes the message posting to be delayed, complicating troubleshooting.

In other words, neither server reports the error properly, and there is no OSC failsafe. Also, without OSC reporting, user code has absolutely no way to detect failure.

We could argue about which commands may not require a reply. I think certainly buffer commands should do it, as they are lower density than for example `/c_set` might be.

Because the documentation is unclear about the spec, I won't say it's a bug (and, accordingly, not for consideration in 3.10).
